### ISSUE TITLE
Support multiple assignments before final print statement

### ISSUE THREAD

**Issue by msayson** [msayson/arithmetic-compiler#1]
Acceptance criteria:
* Compiler successfully processes and interprets multi-line programs with assignments
* Parse and translation tests are added and passing

Support programs with multiple assignments before the terminating print statement.

For example, the following program will output "315" after this task is completed.
```
x = 5 * 3;   // -> 15
y = 100 * 3; // -> 300
"print x + y;"
```

**Comment by msayson** [comment]
Conversion to valid assembly requires #2 (register allocation) to be completed first.

**Comment by msayson** [comment]
Closing as done, fixed with changes in #3.
